Assets/css/style.css

```diff
@@ -463,91 +487,23 @@ article:hover .work-btn {
   color: #242a36;
 }
 
-.submit_btn {
-  border: none;
-  background-color: #f55800;
-  color: white;
-  font-size: 1rem;
-  padding: 12px 18px;
-}
-
 .submit_btn:hover {
   background-color: #b94200;
   cursor: pointer;
 }
 
-/* Setup for mobile devices with screen is 500px */
+/* XXXXXXXXXXXXXXXXXXXXXXXXXXX  mobile devices  XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  */
 
 @media only screen and (max-width: 500px) {
   nav {
     display: none;
   }
 
```

Comment:
I was a big fan of styling for desktop to mobile screens too, but for good convention and good practices, it's almost a must in the modern web development era to start your styles from mobile to desktop screens.

- [ ] Kindly style your website from Mobile to desktop screens as it's required for this project your styles start from mobile to desktop screens and are in the range from 768px and above but not above 1440px.  I suggest you start all your desktop screen styles from 768px above.
- Hint: 
You can put all your desktop screen styles in this media query:
```mdx
@media screen and (min-width: 768px) { ... }
```
and all other styles can be left for mobile screens. 👍